package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "",
+  "private": "true",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
I just learned on [StackOverflow](https://stackoverflow.com/questions/40379139/cannot-find-module-webpack-bin-config-yargs) that for `webpack-cli 4`, we need to use `webpack serve` instead of `webpack-dev-server`.
```JS
"start": "webpack serve --open"
```